vcx/wrappers/java/src/main/java/com/evernym/sdk/vcx/proof/DisclosedProofApi.java

```diff
@@ -123,8 +123,8 @@ public void callback(int commandHandle, int err, String proofRequests) {
 
     private static Callback vcxProofGetStateCB = new Callback() {
         @SuppressWarnings({"unused", "unchecked"})
-        public void callback(int commandHandle, int err, int proofHandle, int state) {
-            logger.debug("callback() called with: commandHandle = [" + commandHandle + "], err = [" + err + "], proofHandle = [" + proofHandle + "], state = [" + state + "]");
+        public void callback(int commandHandle, int err, int state) {
```

Comment:
Note for other reviewers: this seems correct, see the callback definition here
https://github.com/hyperledger/indy-sdk/blob/master/vcx/libvcx/src/api/disclosed_proof.rs#L438